src/main/java/org/jenkinsci/plugins/parallel_test_executor/CountDrivenParallelism.java

```diff
@@ -22,6 +23,7 @@ public int calculate(List<TestClass> tests) {
         return size;
     }
 
+    @Symbol("count")
```

Comment:
Question that just popped to mind - are the `@Symbol`s scoped by anything? Or if we use `count` as the symbol here, does that mean nothing else can use `count` anywhere else? Worrying that we may need some kind of `@Symbol` registry for tracking the things...